Makefile

```diff
@@ -242,7 +248,10 @@ $(objdir)/misc/demangler.o: $(srcdir)/misc/demangler.c $(objdir)/version.h $(COM
 $(objdir)/misc/symbols.o: $(srcdir)/misc/symbols.c $(objdir)/version.h $(COMMON_DEPS)
 	$(QUIET_CC)$(CC) $(SYMBOLS_CFLAGS) -c -o $@ $<
 
-$(filter-out $(objdir)/uftrace.o,$(UFTRACE_OBJS)): $(objdir)/%.o: $(srcdir)/%.c $(COMMON_DEPS)
+$(UFTRACE_OBJS_VERSION): $(objdir)/%.o: $(srcdir)/%.c $(objdir)/version.h $(COMMON_DEPS)
+	$(QUIET_CC)$(CC) $(UFTRACE_CFLAGS) -c -o $@ $<
+
+$(filter-out $(objdir)/uftrace.o $(UFTRACE_OBJS_VERSION), $(UFTRACE_OBJS)): $(objdir)/%.o: $(srcdir)/%.c $(COMMON_DEPS)
```

Comment:
I'm not sure but just adding following line is not enough?

```Makefile
$(UFTRACE_OBJS_VERSION): $(objdir)/version.h
```

Reply:
Yes, it is enough. I tried it and it works. It avoids redundant dependencies and compile command.